# DP: Modify gdc driver to have no libphobos by default.

--- a/src/gcc/d/d-lang.cc
+++ b/src/gcc/d/d-lang.cc
@@ -148,7 +148,7 @@
 d_init_options_struct (struct gcc_options *opts)
 {
   // GCC options
-  opts->x_flag_exceptions = 1;
+  opts->x_flag_exceptions = 0;
 
   // Avoid range issues for complex multiply and divide.
   opts->x_flag_complex_method = 2;
--- a/src/gcc/d/d-spec.c
+++ b/src/gcc/d/d-spec.c
@@ -84,7 +84,7 @@
 
   /* If nonzero, use the standard D runtime library when linking with
      standard libraries. */
-  int phobos = 1;
+  int phobos = 0;
 
   /* The number of arguments being added to what's in argv, other than
      libraries.  We use this to track the number of times we've inserted
